pages/linux/lima.md

```diff
@@ -0,0 +1,8 @@
+# lima
+
+> This command is a wrapper for `limactl`.
+> More information: <https://github.com/lima-vm/lima>.
+
+- View documentation of `limactl`:
+
+`tldr limactl`
```

Comment:
Since this is an alias page, we can use our [generic template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md#en) to facilitate easy translation.

```suggestion
> This command is an alias of `limactl`.
> More information: <https://github.com/lima-vm/lima>.

- View documentation for the original command:

`tldr limactl`
```